pysingfel/gpu/diffraction.py

```diff
@@ -78,6 +93,18 @@ def calculate_diffraction_pattern_gpu(reciprocal_space, particle, return_type='i
         pattern_cos, pattern_sin, atom_type_num, cuda_split_index,
         pixel_number)
 
+    # Add the hydration layer
+    water_position = np.ascontiguousarray(particle.mesh[particle.solvent_mask,:])
+    water_num = np.sum(particle.solvent_mask)
+    water_prefactor = 0.334 * 10**30 * particle.mesh_voxel_size**3
```

Comment:
assign a meaningful variable name (or add comment) for magic number 0.334 * 10**30

Reply:
Done. see commit [62ea779](https://github.com/chuckie82/pysingfel/pull/64/commits/62ea779ea3ec158d8a08923b07ef59918dec08cb)